ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1049,6 +1049,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-kubevirt-hypershift
+- as: aws-ipi-ovn-migration-sdn-p2-f7
+  cron: 58 4 6,14,22,29 * *
+  steps:
+    cluster_profile: aws
```

Comment:
Shall we use `aws-qe` ?

Reply:
@liangxia Good catch! I will update it. 

I always have the problems when I update the code and re commit in release repo, could you help to check if below steps are correct, thanks!
```
sudo make registry-metadata
sudo make ci-operator-config
sudo make jobs
git add openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml
git add openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml
git commit –amend
git push --force-with-lease origin $branchname
```

Reply:
Try this,
```
git fetch upstream master
git rebase upstream/master
# change codes
sudo make update
git add xxx yyy ... zzz
git commit  --amend
git push origin $branchname --force
```